dev/com.ibm.websphere.appserver.features/visibility/public/batchManagement-2.0/io.openliberty.batchManagement-2.0.feature

```diff
@@ -14,10 +14,9 @@ IBM-API-Package: jakarta.batch.api; type="spec", \
  jakarta.batch.runtime.context; type="spec"
 IBM-ShortName: batchManagement-2.0
 Subsystem-Name: Batch Management 2.0
--features=com.ibm.websphere.appserver.servlet-5.0, \
-  io.openliberty.batch-2.0, \
+-features=io.openliberty.batch-2.0, \
```

Comment:
You will need to put servlet-5.0 back in here for resthandler to resolve correctly.